### PR TITLE
[CRCN] Critical Role: Call of the Netherdeep

### PR DESCRIPTION
**Pull request by NickVendel**

### CODE REVIEW

File: supplements/critical-role-call-of-the-netherdeep/items.xml [hunk 1]
```diff
@@ -0,0 +1,624 @@
...
+			<set name="category">Magic Weapons</set>
+			<set name="type" addition="any">Weapon</set>
+			<set name="rarity">Very Rare</set>
+			<set name="weapon">ID_INTERNAL_WEAPON_CATEGORY_SIMPLE_MELEE||ID_INTERNAL_WEAPON_CATEGORY_SIMPLE_RANGED||ID_INTERNAL_WEAPON_CATEGORY_MARTIAL_MELEE||ID_INTERNAL_WEAPON_CATEGORY_MARTIAL_RANGED</set>
+			<set name="name-format">Ruidium {{parent}}</set>
+			<set name="attunement">true</set>
+			<set name="cost" currency="gp">0</set>
+		</setters>
+		<rules>
+			<stat name="innate speed:swim" value="innate speed" bonus="base" alt="Ruidium Shield" />
```
Comment (kgirtxd) [inline supplements/critical-role-call-of-the-netherdeep/items.xml:361]:
```suggestion
			<stat name="innate speed:swim" value="innate speed" bonus="base" alt="Ruidium Weapon" />
```

File: supplements/critical-role-call-of-the-netherdeep/items.xml [hunk 2]
```diff
@@ -0,0 +1,624 @@
...
+			</div>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Legendary</set>
+			<set name="keywords">vestige of divergence, ac, invisibility, reaction, bonus action</set>
+			<set name="attunement">true</set>
+			<set name="charges">7</set>
+			<set name="cost" currency="gp">0</set>
```
Comment (kgirtxd) [inline supplements/critical-role-call-of-the-netherdeep/items.xml:157]:
```suggestion
			<set name="cost" currency="gp">0</set>
			<set name="slot">misc</set>
```
There is a bug with items that require attunement and are not equip-able that sometimes the effect is kept even without the items.  The app also lags more without it

File: supplements/critical-role-call-of-the-netherdeep/items.xml [hunk 3]
```diff
@@ -0,0 +1,624 @@
...
+			</div>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Legendary</set>
+			<set name="keywords">vestige of divergence, ac, invisibility, reaction</set>
+			<set name="attunement">true</set>
+			<set name="charges">5</set>
+			<set name="cost" currency="gp">0</set>
```
Comment (kgirtxd) [inline supplements/critical-role-call-of-the-netherdeep/items.xml:105]:
```suggestion
			<set name="cost" currency="gp">0</set>
			<set name="slot">misc</set>
```
There is a bug with items that require attunement and are not equip-able that sometimes the effect is kept even without the items.  The app also lags more without it

File: supplements/critical-role-call-of-the-netherdeep/items.xml [hunk 4]
```diff
@@ -0,0 +1,624 @@
...
+			</div>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Legendary</set>
+			<set name="keywords">vestige of divergence, ac, invisibility</set>
+			<set name="attunement">true</set>
+			<set name="charges">3</set>
+			<set name="cost" currency="gp">0</set>
```
Comment (kgirtxd) [inline supplements/critical-role-call-of-the-netherdeep/items.xml:64]:
```suggestion
			<set name="cost" currency="gp">0</set>
			<set name="slot">misc</set>
```
There is a bug with items that require attunement and are not equip-able that sometimes the effect is kept even without the items.  The app also lags more without it

**Review by kgirtxd — CHANGES_REQUESTED** [review]

**Review by kgirtxd — APPROVED** [review]

File: supplements/critical-role-call-of-the-netherdeep/items.xml (hunk 1 above)
Comment (AngryGuineaPig) [inline supplements/critical-role-call-of-the-netherdeep/items.xml]: I thought we were going to try and stop using these?

**Review by AngryGuineaPig — APPROVED** [review]

**Pull request merged**